spikeinterface/extractors/nwbextractors.py

```diff
@@ -168,9 +175,9 @@ def get_traces(self, start_frame, end_frame, channel_indices):
         if end_frame is None:
             end_frame = self.get_num_samples()
 
-        with NWBHDF5IO(self._path, 'r') as io:
+        with NWBHDF5IO(self._path, mode='r', load_namespaces=True) as io:
```

Comment:
Why is for this `load_namespaces=True` could you put a very small comments in the code ?

Reply:
for nwbfile that contain custom extensions this argument is necessary. But it works for general files also.

Reply:
OK Sorry. I did not get it.